modules/widgets/contact-info.php

```diff
@@ -33,8 +34,21 @@ function __construct() {
 				$widget_ops
 			);
 			$this->alt_option_name = 'widget_contact_info';
+
+			if ( is_customize_preview() ) {
+				add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
```

Comment:
@westonruter thank you very much for the thorough explanation since it made me understand the functioning. The two examples are great and I'll definitely borrow some inspiration in the future to avoid loading everything at once. For now and given the short time before Apr 12, I agree that it makes sense to do it this way. I've started [making the other widgets compatible](https://github.com/Automattic/jetpack/pull/3607/files) following the approaches for the widgets in this branch. 
Thanks again and have a nice weekend!